frontend/pages/404.tsx

```diff
@@ -0,0 +1,44 @@
+import { Center, Box, Text, Button, Container, Flex } from '@chakra-ui/react';
+import { DEFAULT_STORE_CODE } from '@config/env';
+import { DefaultLayout, DefaultLayoutProps } from '@layouts/default';
+import { getGlobalSettings } from '@models/global-settings';
+import { getStoreByCode, getStoreList } from '@models/store';
+
+export default function Custom404({
+   currentStore,
+   stores,
+   globalSettings,
+}: DefaultLayoutProps) {
+   return (
+      <DefaultLayout {...{ currentStore, stores, globalSettings }}>
+         {' '}
+         {
+            <Center h="56.6vh">
+               <Text
+                  fontSize={20}
+                  fontWeight="bold"
+                  fontFamily="Lato"
+                  padding="16px"
+               >
+                  Error 404: Page not Found
+               </Text>
+            </Center>
+         }
+      </DefaultLayout>
+   );
+}
+
+export async function getStaticProps() {
+   const [globalSettings, stores, currentStore] = await Promise.all([
```

Comment:
you can probably call [`getLayoutServerSideProps`](https://github.com/iFixit/react-commerce/blob/fde3c53162349198a9958dec7bd505c8c1b77518/frontend/layouts/default/getLayoutServerSideProps.ts#L6) here instead of using `Promise.all`. That way, we don't have to keep both `Promise.all`'s up to date.